<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrqXI</class>
 <widget class="QWidget" name="MrqXI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkIsiEn">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cmbIsiType">
     <item>
      <property name="text">
       <string>RESERVE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LOW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RISE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FALL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HIGH</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Reponse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbIsiResp">
     <item>
      <property name="text">
       <string>NONE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALARM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>STOP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALARM+STOP</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Response CH</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbIsiCh">
     <item>
      <property name="text">
       <string>CH1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Period(s)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spinIsiPeriod">
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
